### DISCUSSION TITLE
AdGuard Home Crashing (Container)

### DISCUSSION THREAD

**Discussion by SkiiLLeX** [AdguardTeam/AdGuardHome#5180] [Q&A]
Hello There,

firstly, I would like to describe the situation at my home:
 
I currently have AdGuard running as a Docker container via Portainer. My firewall points to AdGuard as DNS via the DHCP server.
 
My problem which I have with my AdGuard Home:
 
Every 30 minutes or so that AdGuard is entered as the DNS server and traffic is running through AdGuard, AdGuard Home hangs completely away and stops working.

The container is then displayed as "exited".

Unfortunately I can not say why but maybe someone has some idea 

(Log File)
[_AdGuard_logs.txt](https://github.com/AdguardTeam/AdGuardHome/files/10070020/_AdGuard_logs.txt)

**Comment by Joery** [comment]
I seem to have the same issue. Docker container running in a Traefik setup, the container seems to work fine up until I change the DNS in pfSense to AdGuard. Then the container seems to freeze, and the logging simply stops like yours.

**Reply by SkiiLLeX** [reply]
How is your Docker built is that a VM?
I was actually able to resolve the error on my end due to a network type kernel.
